admin/controller/payment/razorpay.php

```diff
@@ -311,6 +311,359 @@ public function uninstall(): void {
 		}
 	}
 
+    /* Subscription Modules */
+    public function getPlan()
+    {
+        $this->load->language('extension/razorpay/payment/razorpay');
+        $this->load->model('extension/razorpay/payment/razorpay');
+
+        if (isset($this->request->get['filter_plan_id']))
+        {
+            $filter_plan_id = trim($this->request->get['filter_plan_id']);
+        }
+        else
+        {
+            $filter_plan_id = '';
+        }
+
+        if (isset($this->request->get['filter_plan_name']))
+        {
+            $filter_plan_name = trim($this->request->get['filter_plan_name']);
+        }
+        else
+        {
+            $filter_plan_name = '';
+        }
+
+        if (isset($this->request->get['filter_plan_status']))
+        {
+            $filter_plan_status = trim($this->request->get['filter_plan_status']);
+        }
+        else
+        {
+            $filter_plan_status = '';
+        }
+
+        if (isset($this->request->get['filter_date_created']))
+        {
+            $filter_date_created = trim($this->request->get['filter_date_created']);
+        }
+        else
+        {
+            $filter_date_created = '';
```

Comment:
@abdulwahidsharief , please define all blank or default value at the start. No need to use else block.